smartseq2_single_sample/labels.json

```diff
@@ -0,0 +1,5 @@
+{
+  "project":"mint-pipeline-dev",
+  "pipeline":"scrnaseq",
```

Comment:
I think you should probably specify ss2 for pipeline instead of scrnaseq, which also includes optimus.